### PR TITLE
Improve makefile of example folder

### PR DESCRIPTION
**Pull request by llamerada-jp**
- use `:=` instead of `=`
- add TMPDIR as a variable
- add some parameters for the test of pvc-autoresizer

Signed-off-by: Yuji Ito <llamerada.jp@gmail.com>

### CODE REVIEW

File: example/Makefile [hunk 1]
```diff
@@ -1,21 +1,22 @@
...
+KIND_CLUSTER_NAME ?= topolvm-example
 
-SUDO=sudo
-BINDIR=$(shell pwd)/bin
-KUSTOMIZE=$(BINDIR)/kustomize
-KUBECTL=$(BINDIR)/kubectl
-KIND=../bin/kind
+SUDO := sudo
+BINDIR := $(shell pwd)/bin
+TMPDIR := /tmp/topolvm
```
Comment (toshipp) [inline example/Makefile:10]: Do you need to use `?=`?

**Review by satoru-takeuchi — APPROVED** [review]
No other comment in addition to Toshi's one.

File: example/Makefile (hunk 1 above)
Comment (llamerada-jp) [reply example/Makefile:10]: I confirmed that no problem caused using `:=` instead of `?=` in this Makefile. I replaced all `?=` to `:=`.

Comment (toshipp) [reply example/Makefile:10]: The behavior is described here:  https://www.gnu.org/software/make/manual/make.html#Overriding

**Review by toshipp — APPROVED** [review]
LGTM

**Review by satoru-takeuchi — APPROVED** [review]

**Pull request merged**